spring-cloud-gcp-starters/spring-cloud-gcp-starter-sql/src/main/java/org/springframework/cloud/gcp/sql/SqlCredentialFactory.java

```diff
@@ -59,7 +59,7 @@ public Credential create() {
 
 		try {
 			return GoogleCredential.fromStream(new FileInputStream(credentialResourceLocation))
-					.createScoped(GcpContextAutoConfiguration.CREDENTIALS_SCOPES_LIST);
+					.createScoped(GcpContextAutoConfiguration.DEFAULT_CREDENTIAL_SCOPES);
```

Comment:
Why not in this instance use only the scopes needed for Cloud SQL?

Reply:
#121 takes care of that, this is just from the refactor.